### ISSUE TITLE
Determine all places the default branch name is referenced in the project

### ISSUE THREAD

**Issue by acrollet** [department-of-veterans-affairs/va.gov-cms#6288]
## Description

As a CMS devops engineer, I need to know all places the default branch name is referenced in the project, so that I can change/rename the default branch without negative effects.

## Acceptance Criteria
- [x] References within code (CMS & devops repos) are documented on this issue
- [x] References within documentation (github & confluence) are documented on this issue
- [x] References within CI configuration (github actions & jenkins) are documented on this issue

**Comment by acrollet** [comment]
(Estimate split from #2010)

**Comment by ndouglas** [comment]
Followups:
- Tugboat
- Views

I created a list of all the appearances of "master" in the CMS codebase, excluding some related projects we don't control (vets-website, devops, etc), etc.  I then went through and pared out some things that were historical (CHANGELOG), irrelevant (data referring to master's degrees and Master Chiefs), or part of something we should follow up with elsewhere (IDK how much control we have over Tugboat, and Views is a separate issue).

[Gist here](https://gist.github.com/ndouglas/033191a8a3aee1e441eaf98e48408a32).

TL;DR: actual mechanical references are few.  It's mostly documentation, and that's easy to fix and low-risk.

I'ma work on the Devops repo now.

**Comment by acrollet** [comment]
Nice work @ndouglas! This should give us a great start on trying this out with cms-test.
